src/components/Class/ClassCard/index.js

```diff
@@ -121,20 +119,10 @@ function ClassCard({ item, handleEnrolledData, handleDropOutData, editClass }) {
           },
         }
       )
-      .then(
-        () => {
-          notify();
-          handleEnrolledData(Id);
-        },
-        (error) => {
-          toast.error(
-            `Something went wrong with error status: ${error.response.status} ${error.response.data.message}`,
-            {
-              position: toast.POSITION.BOTTOM_RIGHT,
-            }
-          );
-        }
-      );
+      .then(() => {
+        notify();
+        dispatch(classActions.enrolledClass(data, index));
```

Comment:
same feedbacks as in the delete commit

Reply:
https://github.com/navgurukul/bhanwari-devi/pull/85/commits/0fe71dd2f5ca43cae07346e91c0c3558e27cf377